src/plugins/passwd/README.md

```diff
@@ -15,7 +15,17 @@ This plugin parses `passwd` files, e.g. `/etc/passwd`.
 
 ## Implementation Details
 
-If present, the not-posix compliant `fgetpwent` function will be used to read the file supplied by the resolver, otherwise `getpwent` will be used. For writing, if present, `putpwent` will be used, if not a simple implementation writing straight to the config file.
+The non-POSIX function `fgetpwent` (GNU_SOURCE) will be used to
+read the file supplied by the resolver.
+As a fallback we implemented our own version which might not behave correctly.
```

Comment:
What does this mean? "might not behave correctly"

Reply:
we do not test for full correctness with our backup implementations. hence there might be hidden issues when those fallbacks are used.

Reply:
Maybe telling that we use the musl implementation is more helpful?